contrib/pyln-testing/pyln/testing/utils.py

```diff
@@ -1693,6 +1693,11 @@ def line_graph(self, num_nodes, fundchannel=True, fundamount=FUNDAMOUNT, wait_fo
         self.join_nodes(nodes, fundchannel, fundamount, wait_for_announce, announce_channels)
         return nodes
 
+    def get_unused_port(self):
+        port = reserve_unused_port()
+        self.reserved_ports.append(port)
+        return port
+
```

Comment:
It's a bit strange to attach this general purpose method to a specific use of it (i.e., nodes being started) but since its a fixture we use everywhere, it doesn't really matter.